src/packet-processors/load-packet-processors.ts

```diff
@@ -80,9 +81,10 @@ export function loadPacketProcessors(): Record<string, PacketProcessor> {
 		eorzeaTimeOffset,
 		equipDisplayFlags,
 		eventPlay,
+		eventPlay32,
 		eventPlay4,
 		eventPlay8,
-		eventPlay32,
+		eventPlayN,
```

Comment:
This file is generated so this is fine, but it'd be nice to exclude this file somehow, since it's only used as a base for the other EventPlay packets.